manage.py

```diff
@@ -0,0 +1,13 @@
+from flask_script import Manager
+from flask_migrate import Migrate, MigrateCommand
+
+
+from src.model import db
+from src.server import server
+
+migrate = Migrate(server, db)
```

Comment:
The variable `migrate` is not using in the code.

Reply:
It is required to use migration commands from manage.py  https://flask-migrate.readthedocs.io/en/latest/#installation

Reply:
OK